pyiron/project.py

```diff
@@ -18,7 +19,7 @@
 from pyiron.vasp.potential import VaspPotential
 import pyiron.atomistics.structure.pyironase as ase
 from pyiron.atomistics.structure.atoms import Atoms
-from pyiron.atomistics.structure.generator import create_surface, create_ase_bulk, create_structure
+from pyiron.atomistics.structure.factory import StructureFactory
```

Comment:
```suggestion
from pyiron.atomistics.structure.factory import StructureFactory
from pyiron_base import Creator as CreatorBase
```

Reply:
@jan-janssen my understanding is that this allows (a) `from pyiron import CreatorBase` and (b) the use of `CreatorBase` freely throughout the pyiron/pyiron module. Is this correct and complete?

If so, I don't understand the purpose here. (a) doesn't seem like something we want people to do, and (b) `CreatorBase` is used only in `pyiron.project.Project` so we don't reduce the number of import lines. Please expand if I'm missing some benefit.

Reply:
I guess you now did the same in line 11 so we do not need it here.